README.md

```diff
@@ -176,6 +176,7 @@ datadreamer --config <path-to-config>
 - `--lm_quantization`: Quantization to use for Mistral language model. Choose between `none` and `4bit`. Default is `none`.
 - `--annotator_size`: Size of the annotator model to use. Choose between `base` and `large`. Default is `base`.
 - `--disable_lm_filter`: Use only a bad word list for profanity filtering. Default is `False`.
+- `--keep_empty_images`: Whether to keep images without any annotations. Default if `False`.
```

Comment:
Isn't this name confusing? Maybe a better name would be something like `keep_unlabeled_images`?

Otherwise, LGTM.

Reply:
That one is better yeah, I'll change it.